sphinx/domains/python.py

```diff
@@ -446,6 +446,25 @@ def get_index_text(self, modname: str, name_cls: Tuple[str, str]) -> str:
 class PyVariable(PyObject):
     """Description of a variable."""
 
+    option_spec = PyObject.option_spec.copy()
+    option_spec.update({
+        'type': directives.unchanged,
+        'value': directives.unchanged,
+    })
```

Comment:
Note: I did not make `:annotation:` option deprecated in this moment. Is it still needed? If not, I'll deprecate it soon.

Reply:
AFAIK, `:annotation:` option is for `autodata` and `autoattribute` directives:
https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoattribute

Ah, I found it.. https://github.com/sphinx-doc/sphinx/blame/edd5fcf8df066f9bd1afb98e2fab04db16ca6f54/sphinx/domains/python.py#L201
but it is [not documented](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html) I think.. 

IMO, it would be better if `annotation` will be a alias of `value` option.

Reply:
Okay, let's keep as is.

Note: `:annotation:` expects the value starts with '=' mark. But `:value:` option does not. So it is not compatible.